dashboard/db/migrate/20210127025620_create_labs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLabs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :labs do |t|
+      t.string :key, null: false
+      t.string :name, null: false
+      t.text :properties
+
```

Comment:
You might want to add a line like [this](https://github.com/code-dot-org/code-dot-org/blob/d4c424dae86733abf19e6d4101444d9e09fffa47/dashboard/db/migrate/20201117160845_create_user_ml_models.rb#L14), even though your schema.rb change looks correct. Migrations that create tables have been problematic from a utf8/utf8mb4 perspective lately